cli/internal/cmd/configfetchmeasurements.go

```diff
@@ -79,6 +88,46 @@ func configFetchMeasurements(cmd *cobra.Command, fileHandler file.Handler, clien
 	return nil
 }
 
+func verifyWithRekor(cmd *cobra.Command, hash string) error {
+	r, err := sigstore.NewRekor()
+	if err != nil {
+		cmd.PrintErrln("Unable to construct Rekor client.")
+		return err
+	}
+
+	uuids, err := r.SearchByHash(cmd.Context(), hash)
+	if err != nil {
+		cmd.PrintErrln("Unable to find hash in Rekor.")
+		return err
+	}
+
+	// We expect the first entry in Rekor to be our original entry.
+	// SHA256 should ensure there is no entry with the same hash.
+	// Any subsequent hashes are treated as potential attacks and are ignored.
+	artifactUUID := uuids[0]
```

Comment:
Should we add a warning then to make the attack visiable, or do we not need to expose it to the user since all attacks are mitigated through choosing the first entry?

Reply:
The proposed action is to watch the transparency log for attacks. This should be done by a service or daemon running on our backend, e.g., https://github.com/nsmith5/rekor-sidekick